### DISCUSSION TITLE
I have submitted my application 16 times

### DISCUSSION THREAD

**Discussion by Wangyiheng962** [community/community#50987] [GitHub Education]
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

**Comment by BiscuitFingers** [comment]
I am getting the same issue

**Comment by vladknezdata** [comment]
Same issue. There is no upload button to upload the better looking docs. I updated the profile, photographed the student ID and the transcripts, both on a new iMac, on an iPhone, I use my school email, I'm working on the school project with other from school right now. Project is initiated and shared by a school professor. Location is also an issue since I'm not on campus. There are no documents issued by school that state anyone as a distance learner that I am aware of. Full time student at ASU, receiving student aid. What else can I do? Come by to GitHub HQ? I live in SF, I can do that as well.

**Comment by samus-aran** [comment]
Hi @Wangyiheng962 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
